tests/global/test_cli.py

```diff
@@ -6,6 +6,6 @@
 class EntryPointTest(ErtTest):
     def test_single_realization(self):
         config_file = self.createTestPath('local/poly_example/poly.ert')
-        exec_path = os.path.join(self.SOURCE_ROOT, "ert_gui/bin/ert_cli")
-        ok = subprocess.call([exec_path, config_file, "test_run", "default"])
+        exec_path = os.path.join(self.SOURCE_ROOT, "bin/ert")
+        ok = subprocess.call([exec_path, "cli", config_file, "--mode", "test_run", "--target-case", "default"])
```

Comment:
When you are touching this anyway, I think the last two lines could be replaces by a `subprocess.checkcall`